src/content/glossary/index.md

```diff
@@ -644,15 +644,15 @@ The third development stage of Ethereum, launched in October 2017.
 
 ### mining {#mining}
 
-The process of verifying transactions and contract execution on the Ethereum blockchain in exchange for a reward in ether with the mining of every block.
+The process of verifying transactions and contract execution on the Ethereum blockchain in exchange for a reward in ether with the mining of every block. This was how Ethereumw as secured before it moved to [proof-of-stake](#pos).
 
 ### mining pool {#mining-pool}
```

Comment:
Do we still need this in glossary?

Reply:
I guess ignore these comments for now. Maybe as a post-merge clean-up, we can discuss the possibility of removing outdated lingo from the glossary.

Reply:
I think we should keep it. People will definitely be confused about what happened to mining and where it fits in Ethereum for a long time after TM

Reply:
Sorry, my comment was specifically about mining pools, but your point. still seems valid.